### PR TITLE
Camunda 8.5 generation name scheme announcement

### PR DESCRIPTION
**Pull request by megglos**
## Description

Relates to https://github.com/camunda/engineering-leadership-team/issues/76

## When should this change go live?

Ideally asap, so I can refer to it when also communicating with other departments within Camunda, e.g. support.

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by megglos** [comment]
what's this I only touched the announcements file :D

**Comment by akeller** [comment]
> what's this I only touched the announcements file :D

Updating the branch should resolve this. I just hit the button so it should look more reasonable now.

@megglos do you want this out today with the alpha or with 8.5 next month?

**Comment by megglos** [comment]
> @megglos do you want this out today with the alpha or with 8.5 next month?

not relevant for the alpha only for 8.5.0

**Review by MaxTru — APPROVED** [review]
👍

File: docs/reference/announcements.md [hunk 1]
```diff
@@ -4,6 +4,30 @@ title: "Announcements"
...
 
+## Camunda 8.5
+
+Release date: 9th of April 2024
+
+End of maintenance: 14th of October 2025
+
+:::info
+
+### Camunda Cloud: New Generation naming scheme
```
Comment (MaxTru) [inline docs/reference/announcements.md:15]:
```suggestion
### Camunda SaaS: New Generation naming scheme
```

AFAIK `SaaS`is the term we should use (compare https://camunda.com/pricing/). But this is also a question for @christinaausley  / DevEx :-)

**Review by MaxTru — CHANGES_REQUESTED** [review]

File: docs/reference/announcements.md (hunk 1 above)
Comment (christinaausley) [reply docs/reference/announcements.md:15]: Camunda 8 SaaS/ Camunda 8 Self-Managed is the preference here.

**Comment by megglos** [comment]
@MaxTru @christinaausley applied suggested change, ready for another 👀

**Review by MaxTru — DISMISSED** [review]

**Comment by akeller** [comment]
Non-blocking for this PR! @megglos @christinaausley 

This information should live outside the announcements page, but it's good to announce it here because it is a change we want to make sure people notice and understand. For the scope of this PR, this makes sense.

Long term, we should add similar info to our [release policy page](https://docs.camunda.io/docs/reference/release-policy/).

You may also want to add something to https://camunda.com/release-policy/.

Please create a follow-up issue so we don't lose this context on the announcements page.

**Comment by MaxTru** [comment]
@christinaausley can you please have an eye on this PR and ensure we have it merged and released for 8.5 latest?

**Comment by MaxTru** [comment]
Actually, why do we not merge this PR right away? @megglos

**Review by christinaausley — APPROVED** [review]

**Pull request merged**